src/main/java/Alpa.java

```diff
@@ -0,0 +1,284 @@
+    printDecoratedMessage(message.toString());
+  }
+
+  private static void loadTasks() {
+    File file = new File("./data/tasks.txt");
+    if (file.exists()) {
+      try (Scanner scanner = new Scanner(file)) {
+        while (scanner.hasNext()) {
+          String line = scanner.nextLine();
+          String[] parts = line.split(" \\| ");
+          if (parts.length < 3) {
+            continue; // Invalid line format, skipped
+          }
+          TaskType taskType = TaskType.fromShortName(parts[0]);
+          if (taskType == null) {
+            continue; // Skip if task type is unknown
+          }
+          Task task;
+          switch (taskType) {
+          case TODO:
+            task = new ToDo(parts[2]);
+            break;
+          case DEADLINE:
+            if (parts.length < 4) {
+              continue; // Additional check for Deadline and Event
+            }
+            task = new Deadline(parts[2], parts[3]);
+            break;
+          case EVENT:
+            if (parts.length < 4) {
+              continue;
+            }
+            String[] times = parts[3].split(" - ");
+            if (times.length < 2) {
+              continue;
+            }
+            task = new Event(parts[2].trim(), times[0].trim(), times[1].trim());
+            break;
+          default:
+            continue;
+          }
+          if ("1".equals(parts[1])) {
+            task.markAsDone();
+          }
+          tasks.add(task);
+        }
+      } catch (FileNotFoundException e) {
+        e.printStackTrace();
+        printDecoratedMessage("File not found human..." + e.getMessage());
+      } catch (Exception e) {
+        e.printStackTrace();
+        printDecoratedMessage("Error reading from file..." + e.getMessage());
+      }
+    } else {
+      new File("./data").mkdirs();
+    }
+  }
+
+  private static void saveTasks() {
+    try (FileWriter fw = new FileWriter("./data/tasks.txt");
+        BufferedWriter bw = new BufferedWriter(fw);
+        PrintWriter out = new PrintWriter(bw)) {
+      for (Task task : tasks) {
+        out.println(task.toFileFormat());
+      }
+    } catch (IOException e) {
+      e.printStackTrace();
+      printDecoratedMessage("\nError! Could not save tasks!" + e.getMessage());
+    }
+  }
+
+  private static void handleBye() {
+    System.out.println("\nIt's been a pleasure grazing through your tasks! Goodbye human! Stay cozy!\n");
+    System.out.println("𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣𖥧𖡼.𖤣\n");
+  }
+
+  private static void handleList() {
+    if (tasks.isEmpty()) {
+      printDecoratedMessage("\nYour list is empty, human!");
+      return;
+    }
+    StringBuilder listOutput = new StringBuilder("\nYour list, human!\n");
+    for (int i = 0; i < tasks.size(); i++) {
+      listOutput.append("  ").append(i + 1).append(". ").append(tasks.get(i)).append("\n");
+    }
+    printDecoratedMessage(listOutput.toString());
+  }
+
```

Comment:
I like how the namings are intuitive, making it easier for me to understand the method that you are calling.